cmd/ncproxy/ncproxy.go

```diff
@@ -557,6 +558,58 @@ func (s *grpcService) GetNetworks(ctx context.Context, req *ncproxygrpc.GetNetwo
 // client connections.
 type ttrpcService struct {
 	m sync.Mutex
```

Comment:
It's unclear what this lock controls access to. Can the name or a comment clarify this?

Reply:
Could be in a separate PR if it's unrelated to this

Reply:
I don't mind adding it here.